archive/android_bootldr.ksy

```diff
@@ -0,0 +1,36 @@
+meta:
+  id: android_bootldr
```

Comment:
The file name and `/meta/id` should include some identifier of environment for which this format is specific, because I guess this is not anything like the "main" Android `bootloader.img` format or superior to all other ones. After all, you yourself described another variation of the `bootloader.img` format for another class of Android devices in https://github.com/kaitai-io/kaitai_struct_formats/pull/472.

Reply:
I have renamed this. Please see if the new name is acceptable.